CDS_Dashboard/script.js

```diff
@@ -353,320 +346,145 @@ chart = new Highcharts.Chart({
-                                                'categories': categories,
-                                                'data': data,
-                                                'type': chartType
-                                            // return options;
-                                        };
-                                    }
-                                    setSecondChart(options);
-                                }
-                            } //end of responsive else statement
-                        }
-
-                    } //end of click
-                } // end of events
-            }
-        } //end of series
-    }, //end of plotOptions
-    tooltip: {
-        formatter: function () {
-            var point = this.point, // sets keyword 'this' -- this.point is really series.data
-                s = point.name + ': ' + point.list + '. '; //formats pointer
-            if (point.deepDrill) {
-                s = point.name + '<br/>' + 
-                    'Services: ' + point.services + '<br/>' +
-                    'Objective: ' + point.objective + '<br/>' + '<br/>' +
-                    '<em>' + 'Click to return home' + '</em>';
-            } else if (point.objective == 'Numerous Objectives') { 
-                s = point.name + '<br/>' + 
-                    '<em>' + 'Click Here to Learn More About ' + point.name + ' Projects' + '</em>';
-            } else if (point.drilldown){
-                s = 'Click to view ' + point.name;
-            } else {
-                s = point.name + '<br/>' + 
-                    'Initiative: ' + point.initiative + '<br/>' +
-                    'Services: ' + point.services + '<br/>' +
-                    'Objective: ' + point.objective + '<br/>' + '<br/>' +
-                    '<em>' + 'Click to Learn More' + '</em>';
-            }
-            return s;
         }
-    },
-    series: [{  // sets chart series here 
-        type: chartType,
-        name: name,
-        data: data,
-        color: 'white' // don't know why you set this to white
-    }],
+    }, //end of plotOptions
     exporting: {
         enabled: false // can't print chart
     }
-}); // end of chart = new Highcharts.Chart()
-
+});
 
-
-chart2 = new Highcharts.Chart({
-    credits: {
-        enabled: false
-    },
-    chart: {
-        style: {
-            fontFamily: 'SourceSansProRegular, Open Sans, Helvetica Neue, Arial, sans-serif'
+    var chart1 = {
+        chart: {
+            renderTo: 'chartContainer'
         },
-        renderTo: 'secondContainer', // passing it where to put it on the HTML - there is a div with ID 'chartContainer'
+        plotOptions: {
+            series: {
+                cursor: 'pointer', // cursor set to pointer since we have 'click' events
+                point: {
+                    events: {
+                        click: function () {
+                            var drilldown = this.drilldown; // uses keyword 'this'
```

Comment:
I figured that it makes more sense semantically. I could technically pass this.drilldown to responsiveHighCharts and make it take a second parameter but it seems like 'good practice' to declare 'this' variables (that's what they said in my JS class at least) - similar to why people use "var self = this" in code.